elements/pfe-tabs/test/pfe-tabs_test.js

```diff
@@ -1,5 +1,23 @@
 /* @TODO: Add focus-state tests to validate against document.activeElement */
 
+// Themes and their expected hex values
+const background_colors = {
```

Comment:
These are covered now by visual regression testing (they snapshot the demo pages) so as long as this appears on the demo page, the colors are being validated.